source/_integrations/wallbox.markdown

```diff
@@ -32,3 +33,10 @@ The integration adds the following sensors:
 - Max Available Power
 - State of Charge
 - Status Description
+- Max Charging Current
+
+## Number
+
+The integration adds the following number (input) component:
```

Comment:
`input` is for manual helpers, component is a term we do not use for user-facing documentation.

Suggestion:

```suggestion
The integration adds the following number entity:
```

Reply:
I changed it into entity.